plutus-core/plutus-ir/src/PlutusIR/Analysis/Builtins.hs

```diff
@@ -41,15 +48,18 @@ makeLenses ''BuiltinMatcherLike
 
 -- | All non-static information about builtins that the compiler might want.
 data BuiltinsInfo (uni :: Type -> Type) fun = BuiltinsInfo
-  { _biSemanticsVariant :: PLC.BuiltinSemanticsVariant fun
-  , _biMatcherLike      :: Map.Map fun (BuiltinMatcherLike uni fun)
+  { _biSemanticsVariant        :: PLC.BuiltinSemanticsVariant fun
+  , _biMatcherLike             :: Map.Map fun (BuiltinMatcherLike uni fun)
+  -- See Note [Unserializable constants]
+  , _biUnserializableConstants :: Some (ValueOf uni) -> Bool
```

Comment:
Do we care about -ize versus  -ise?  Probably not:  I think we're pretty inconsistent elsewhere.  You could go for broke and call them "unzerializable" though.

Reply:
We're massively inconsistent. I'm inconsistent in my own writing. I have no idea which is which, I just use them randomly.